components/02-elements/breadcrumbs/_breadcrumbs.scss

```diff
@@ -1,33 +1,56 @@
-$breadcrumbs-padding        : 10px;
-$breadcrumbs-margin         : 10px;
-$breadcrumbs-color          : $font-color-base; 
-$breadcrumbs-color-second   : $gray;
-$breadcrumbs-clickable-area : 48px;
+$breadcrumbs-margin         : 8px !default;
+$breadcrumbs-padding        : 8px !default;
+$breadcrumbs-clickable-area : 48px !default;
```

Comment:
I get the deal, that should be 48px height... but... when width of the screen is smaller and the breadcrumbs are in two lines, the space between the line is too large. What do you think @kruchy8 @Igloczek

Reply:
I have consulted it with Michal and the decision was to leave it with 48px.